src/phantasm-renderer/backend/Backend.hh

```diff
@@ -39,12 +39,15 @@ public:
     /// causes an internal resize
     virtual void onResize(tg::isize2 size) = 0;
 
-    /// gets the current backbuffer size
+    /// returns the current backbuffer size
     [[nodiscard]] virtual tg::isize2 getBackbufferSize() const = 0;
```

Comment:
hm I feel like adding [[nodiscard]] for EVERY non-void method is overkill and introduces way more noise into the code than necessary. maybe we should only add it when the return value is important but the method name does not make it clear? I mean for `getXYZ` it is obvious that you need the return value but for `clearPendingSize()` it totally is not.

Reply:
I agree, i removed them for the obvious getters.